src/Composer/Exception/MultipleComposerJsonException.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/*
+ * This file is part of Composer.
+ *
+ * (c) Nils Adermann <naderman@naderman.de>
+ *     Jordi Boggiano <j.boggiano@seld.be>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Composer\Exception;
+
+/**
+ * @author Wissem Riahi <wissemr@gmail.com>
+ */
+class MultipleComposerJsonException  extends \RuntimeException
```

Comment:
I think this should be a simple RuntimeException, no need for extra exception types for such minor feature IMO. And especially as we have different exception types, the exception name here does not always make sense.